R/aaa-auto.R

```diff
@@ -8173,7 +8175,7 @@ fundamental_cycles_impl <- function(
   res <- .Call(
     R_igraph_fundamental_cycles,
     graph,
-    start - 1,
+    if (!is.null(start)) start - 1 else NULL,
```

Comment:
I think this logic should leave outside of the call. Above in `if (!is.null(start)) {` also already make `start <- start - 1` and here in `.Call` just have `start - 1` (I understand this means changing the Stimulus config file).